h/static/scripts/directive/share-dialog.coffee

```diff
@@ -0,0 +1,29 @@
+###*
```

Comment:
This file is inconsistent with the rest of our CoffeeScript. It has four space indents instead of two.